Library/Homebrew/diagnostic.rb

```diff
@@ -443,6 +443,32 @@ def check_access_cellar
         EOS
       end
 
+      def check_for_empty_taps
+        taps_dir = HOMEBREW_LIBRARY/"Taps"
+        return unless taps_dir.exist?
+        return unless taps_dir.directory?
+
+        found_empty_dirs = []
+
+        # Warn USER of empty taps_dir children directories
+        taps_dir.children.select{ |c| c.directory? } \
```

Comment:
Okay, that makes sense. However, if I create some empty directories in the `$(brew --prefix)/Library/Taps` folder with `mkdir`, `Tap.map(&:path)` doesn't report them. At least from the `brew irb` instance.